test/acceptance/features/accounting/replace_eu_fund_code_fdmr.feature

```diff
@@ -0,0 +1,34 @@
+Feature: Dual Accounting
+
+    As a payment service
+    I want the EU fund code to be applied to the payment request for FDMR, if applicable
```

Comment:
It's the EU fund code we're replacing, we want the code to be changed to a domestic fund code.